src/common/isolatable_hardwares.cpp

```diff
@@ -1100,6 +1083,17 @@ IsItIsoHwInvPath itemPrettyName(sdbusplus::bus::bus& bus,
     {
         auto retPrettyName = utils::getDBusPropertyVal<std::string>(
             bus, objPath, "xyz.openbmc_project.Inventory.Item", "PrettyName");
+        if (std::get<std::string>(prettyName) == "OpenCAPI Memory Buffer")
+            return retPrettyName == "OpenCAPI Memory Buffer 2A" ||
```

Comment:
Just for readability, I would add brackets

return ( (retPrettyName == "Onboard Memory Power Management IC for 2A and 2B") ||
             (retPrettyName == "Onboard Memory Power Management IC") )